Export-GithubIssues-To-Spreadsheet/GithubToSpreadsheet/src/main/synapse-config/sequences/githubToSpreadsheet.xml

```diff
@@ -0,0 +1,143 @@
+			     mc.setProperty("github.accessToken",java.lang.System.getenv("github_accessToken"));
+			     mc.setProperty("github.mediaType",java.lang.System.getenv("github_mediaType"));
+			     mc.setProperty("github.owner",java.lang.System.getenv("github_owner"));			     
+			     mc.setProperty("github.repos", java.lang.System.getenv("github_repos"));			     
+			     mc.setProperty("github.milestone",java.lang.System.getenv("github_milestone"));
+			     mc.setProperty("github.state",java.lang.System.getenv("github_state"));
+			     mc.setProperty("github.assignee",java.lang.System.getenv("github_assignee"));
+			     mc.setProperty("github.creator",java.lang.System.getenv("github_creator"));
+			     mc.setProperty("github.mentioned",java.lang.System.getenv("github_mentioned"));
+			     mc.setProperty("github.sort",java.lang.System.getenv("github_sort"));
+			     mc.setProperty("github.direction",java.lang.System.getenv("github_direction"));			     
+			     mc.setProperty("github.labels",java.lang.System.getenv("github_labels"));
+			     mc.setProperty("github.perPage",java.lang.System.getenv("github_perPage"));
+			     mc.setProperty("github.since", mc.getProperty("githubToSpreadsheetRegistryTimestamp"));
+			     mc.setProperty("googlespreadsheet.accessToken",java.lang.System.getenv("googlespreadsheet_accessToken"));
+			     mc.setProperty("googlespreadsheet.refreshToken",java.lang.System.getenv("googlespreadsheet_refreshToken"));
+			     mc.setProperty("googlespreadsheet.clientId",java.lang.System.getenv("googlespreadsheet_clientId"));
+			     mc.setProperty("googlespreadsheet.clientSecret",java.lang.System.getenv("googlespreadsheet_clientSecret"));
+			     mc.setProperty("googlespreadsheet.apiUrl",java.lang.System.getenv("googlespreadsheet_apiUrl"));
+			     mc.setProperty("googlespreadsheet.spreadsheetId",java.lang.System.getenv("googlespreadsheet_spreadsheetId"));
+			     mc.setProperty("googlespreadsheet.range",java.lang.System.getenv("googlespreadsheet_range"));
+			     mc.setProperty("googlespreadsheet.valueInputOption",java.lang.System.getenv("googlespreadsheet_valueInputOption"));			     
+			    
+ 			     var dt = new Date();
+			     var time = dt.toISOString();
+			     mc.setProperty("newsystemTime",time);]]></script>
+    <script language="js"><![CDATA[var payloadXML = new XML(<root/>);
+        for each (var repo in String(mc.getProperty("github.repos")).split(',')) {
+            payloadXML.appendChild(new XML(<repo>{repo}</repo>));
+        }
+        mc.setPayloadXML(payloadXML);]]></script>
+    <log level="custom">
+        <property expression="get-property('newsystemTime')" name="NewSystemTime**************************" xmlns:ns="http://org.apache.synapse/xsd"/>
+    </log>
+    <property name="length" scope="operation" type="STRING" value=""/>
+    <iterate continueParent="true" expression="//repo" id="repo_iterator" sequential="true" xmlns:ns="http://org.apache.synapse/xsd">
```

Comment:
Try for each mediator instead of iterator mediator here.

Reply:
If we use for each mediator, we can't use Call, Send and Callout mediators within for each mediator.